src/components/Popup/Popup.tsx

```diff
@@ -135,7 +138,9 @@ export default class Popup extends AutoControlledComponent<Extendable<IPopupProp
             ...accessibility.keyHandlers.trigger,
           })}
         </Ref>
-        {open && this.renderPopupContent(rtl, accessibility)}
+        {open &&
+          Popup.isBrowserContext &&
+          createPortal(this.renderPopupContent(rtl, accessibility), document.body)}
```

Comment:
This is probably more related to #282 but what's the reason to use React Portal directly instead of using stardust Portal component?
The main benefit of using stardust Portal component would be a reuse of `onOutsideClick` (close popup when user clicks outside the popup/trigger) and similar.

Reply:
would rather like to address it by means of separate PR. The reason for that is, primarily, the following: apart from introducing this common functionality, the `Portal` component introduces quite a few additional conventions, such as `render as direct body's children`, etc - and there are not intuitive for the client given just the `Portal`'s name and API - probably, it should be renamed first.

So, my intuition is to introduce the functionality that is necessary as a first thing, and after that generalize by making refactoring of `Portal` component - in a way of making this component itself more generic and properly translate its functionality and application strategies via its name and API (`BodyPortal` is something that comes to my head right now, but again, would rather like to see a clear picture first).

Please, let me know about your thoughts on that.

Reply:
If these changes are unblocking some other pull request, I am okay with them being merged and then the Portal can be replace with Stardust's Portal. In that way we can make progress on both aspects.

Reply:
have discussed with @sophieH29 - essentially, have agreed on the same plan. Lets wait on @miroslavstastny's comments and merge if there won't be anything critical.

Reply:
Go for now.